bulk_deploy/tsuru_api_service.rb

```diff
@@ -0,0 +1,119 @@
+require_relative './../dsl/ssh_helper'
+
+class TsuruAPIService
+
+  def initialize(logger:, api_client:, tsuru_home:)
```

Comment:
Works for me. Just running it against danjim environment. I will wait for the script to finish but it seems to be working.

What is your Ruby version?